### PR TITLE
Crud venue & view details

### PR DESCRIPTION
**Pull request by JonasJallet**
add CRUD venue
Delete items views and manager/controller
Update all views with previous data on edit 
fix update for all crud

### CODE REVIEW

File: src/Controller/ArtistController.php [hunk 1]
```diff
@@ -35,32 +35,23 @@ public function edit(int $id): ?string
...
-            }
-
-            if (!isset($artist['image']) || empty($artist['image'])) {
-                $errors[] = "L'image doit être renseignée";
-            }
+            $artistManager->artistFieldEmpty($artist);
 
-            if (count($errors) === 0) {
+            $errors = $artistManager->getCheckErrors();
+            if (empty($artistManager->getCheckErrors())) {
```
Comment (aldaitzwild) [inline src/Controller/ArtistController.php]: Why do you call getCheckErrors again instead of using $errors ?

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/Controller/ArtistController.php (hunk 1 above)
Comment (JonasJallet) [reply src/Controller/ArtistController.php]: fix $error variable

File: src/View/Concert/_form.html.twig [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
-				<label for="place">Salle</label>
-				<input class="form-control" type="text" value="{{ concert.place }}" name="place" id="place" aria-describedby="champ de saisi salle" placeholder="Rentrez un nom de salle" required>
-			</div>
-
-			<div class="mb-3 form-label">
-				<label for="city">Ville</label>
-				<input class="form-control" type="text" value="{{ concert.city }}" name="city" id="city" aria-describedby="champ de saisi ville" placeholder="Rentrez une ville" required>
+			<div class="mb-3 mt-3 form-label">
+				<label for="artist-select">Sélectionnez un lieu :</label>
+				<select class="form-select" aria-describedby="menu déroullant séléction salle" name="venue_id" id="venue_id" required>
```
Comment (aldaitzwild) [inline src/View/Concert/_form.html.twig]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby

VS

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Concert/_form.html.twig (hunk 2 above)
Comment (JonasJallet) [reply src/View/Concert/_form.html.twig]: update form : change aria-describedby with aria-description

File: src/View/Venue/_form.html.twig [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<form method="post" aria-description="formulaire ajout de lieux">
+	<input type="hidden" value="{{ venue.id }}" name="id">
+	<div class="addvenues row justify-content-center form-floating my-2">
```
Comment (aldaitzwild) [inline src/View/Venue/_form.html.twig]: The class "addvenues" doesn't seem to exists

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Venue/_form.html.twig (hunk 3 above)
Comment (JonasJallet) [reply src/View/Venue/_form.html.twig]: delete useless class

**Review by aldaitzwild — APPROVED** [review]
WORST PR EVER !

**Pull request merged**